### PR TITLE
Add utils to normalize repo

### PR DESCRIPTION
**Pull request by shiyu22**
Signed-off-by: shiyu22 <shiyu.chen@zilliz.com>

### CODE REVIEW

File: tests/unittests/utils/test_repo_normalize.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    def test_parse_uri(self):
+        result = rn1.parse_uri()
+        self.assertEqual(result.full_uri, 'https://towhee.io/towhee/resnet-image-embedding?ref=my-tag&model=resnet50#operator')
+        self.assertEqual(result.author, 'towhee')
+        self.assertEqual(result.repo, 'resnet-image-embedding')
+        self.assertEqual(result.ref, 'my-tag')
+        self.assertEqual(result.repo_type, 'operator')
+        self.assertEqual(result.scheme, 'https')
+        self.assertEqual(result.netloc, 'towhee.io')
+        self.assertEqual(result.query, {'model': 'resnet50'})
```
Comment (reiase) [inline tests/unittests/utils/test_repo_normalize.py:39]: the following properties should be added:
1. result.module_name
2. result.class_name
3. result.function_name

Comment (shiyu22) [reply tests/unittests/utils/test_repo_normalize.py:39]: What's the function_name? And is there any format for module_name?

**Comment by shiyu22** [comment]
#550

**Review by reiase — APPROVED** [review]

**Pull request merged**